adam/learner/preposition_subset.py

```diff
@@ -396,13 +393,5 @@ def replace_template_variables_with_object_names(
                         1.0,
                     )
                 )
-                if debug_mapping:
```

Comment:
re: this deletion - note this would never be reached if the match were in fact partial, because if it were `.matches()` would return nothing.